tests/integrations/virtual_environment/test_utils.py

```diff
@@ -0,0 +1,28 @@
+ENVIRONMENT_TEST_PARAMS = [
+    (None, {}),
+    ({"CUSTOM": "value"}, {}),
+    (None, {"cwd": "/tmp", "timeout": 30}),
+    ({"PATH": "/custom"}, {"cwd": "/tmp", "check": True}),
+]
+
+
+def assert_environment_handling(
+    mock_full_env, env_override, mock_method, method_args, other_kwargs
```

Comment:
Preferred style is one argument per line; ruff/black will apply this automatically if you add a comma after `other_kwargs`.